src/pheval/__init__.py

```diff
@@ -1,5 +0,0 @@
-"""
```

Comment:
Are you sure you want to delete the init file?

Reply:
The init file won't be deleted.
I just erased its content because during the docs generation, empty files will be ignored